src/home/components/Form/index.stories.tsx

```diff
@@ -5,18 +5,29 @@ import { Flex } from '@olist/united';
 import Form, { FormProps } from '~/home/components/Form';
 
 // eslint-disable-next-line react/prop-types
-export const Example = (props: FormProps): ReactElement => {
+export const WithoutError = (props: FormProps): ReactElement => {
```

Comment:
Acho que tem como remover esse disable da regra eslint acima se for feito algo assim:

```
export const WithoutError: React.FC<FormProps> = (props: FormProps) => {
  ...
```

Reply:
Na vdd curiosamente não precisava do disable hehe